.conform.yaml

```diff
@@ -7,23 +7,26 @@ policies:
         case: lower
         invalidLastCharacters: .
       body:
-        required: true
+        required: false
```

Comment:
Question: Dependabot doesn't create commit bodies?

Reply:
Dependabot includes commit bodies. This is more of a me change. I don't create commit bodies so this exact commit would have failed without this change. Thoughts?

Reply:
It looks to me like Dependabot creates well-formed commit bodies with a DCO and everything. Is this unrelated?

Reply:
Correct. I can split this out if requested.

Reply:
Yeah, there's probably not a good reason for us to enforce commit bodies at the moment.